webpack.common.js

```diff
@@ -131,14 +131,33 @@ module.exports = {
   module: {
     rules: [
       {
-        test: /\.js(x?)$/,
+        test: /\.(js|ts)(x?)$/,
         exclude: /node_modules/,
-        use: ['babel-loader'],
-      },
-      {
-        test: /\.ts(x?)$/,
-        exclude: /node_modules/,
-        use: ['babel-loader'],
+        use: {
+          loader: 'swc-loader',
+          options: {
+            jsc: {
+              target: 'es2015',
+              parser: {
+                syntax: 'typescript',
+                tsx: true,
+                decorators: true,
+                dynamicImport: true,
+              },
+              transform: {
+                legacyDecorator: true,
+                decoratorMetadata: true,
+                react: {
+                  runtime: 'automatic',
+                },
+              },
+              externalHelpers: true,
+            },
+            env: {
+              targets: '> 0.25%, not dead',
```

Comment:
Now only including features and polyfills necessary for browser versions used by more than 0.25% of the global population

Reply:
How does this effect the JS size?

Reply:
Less polyfills = less code necessary to be shipped

Older browsers don't have support for all the recent JS features, so they need to have polyfills (code that implements features that aren't supported). This adds weight to the shipped bundle